### ISSUE TITLE
Icon button tap zones are too small or unresponsive

### ISSUE THREAD

**Issue by diegoarff** [heroui-inc/heroui-native#90]
### Before submitting a new issue

- [x] I tested using the latest version of the library, as the bug might be already fixed.
- [x] I tested using a [supported version](https://github.com/reactwg/react-native-releases/blob/main/docs/support.md) of react native.
- [x] I checked for possible duplicate issues, with possible answers.

### Bug summary

Hello! This is another report using the demo app, as I unfortunately cannot test directly with the lib right now. 

Testing on an Android device using the Expo Go demo app, there are some icon buttons like the theme toggle and the internal navigation button that seem to have very small tap zones or unresponsive buttons altogether, leading to a very bad user experience where the user needs to tap multiple times until the button finally triggers.

Here's some videos showing tap input to better demonstrate the issue:

https://github.com/user-attachments/assets/1a3757c9-a205-495a-a0b4-9c15c25d899c

https://github.com/user-attachments/assets/65214cfa-f855-4a9f-873a-53cdfa6f3e7e

### Library version

Used in demo app

### Environment info

```shell
Demo app in Expo Go
```

### Steps to reproduce

1. Open demo app on Expo Go in an Android device
2. Use theme toggle or another icon button

### Reproducible example repository

Demo app

**Comment by Ikhan** [comment]
Tested from iOS as well(iphone 13mini) I have had this issue as well, there is a unresponsive button trigger needs to be clicked multiple times to get it work.

**Comment by villAsh** [comment]
Can I work on this?

**Comment by BoweFrankema** [comment]
@vvv-sss when testing this on Android (Pixel 9) using the Expo GO link on beta 4, I'm still unable to tap on the theme switch toggle.. It could be that the theme switcher itself does not work, but in any case it does not respond to taps. 

Fantastic work on this btw, looks incredible!
